components/FormCard.test.js

```diff
@@ -88,6 +88,19 @@ describe('FormCard Component', () => {
     expect(container).toMatchSnapshot()
   })
 
+  test('Should display submit error message', () => {
+    const mockError = 'Helpful Error Message Here :)'
+    const { queryByText } = render(
+      <FormCard
+        onChange={() => {}}
+        values={[{ title: '' }]}
+        submitError={mockError}
+        onSubmit={mockBtn}
+      />
+    )
+    expect(queryByText(mockError)).toBeTruthy()
```

Comment:
FYI, you could just check so it's not null. That would be more explicit.

Reply:
Or use `jest-dom` matchers

```js
expect(queryByText(mockError)).toBeInTheDocument()
// OR
expect(queryByText(mockError)).toBeVisible()
```

Reply:
Added jest-dom and switched to `toBeVisible()` matcher